src/flash/com/longtailvideo/jwplayer/media/VideoMediaProvider.as

```diff
@@ -90,11 +90,8 @@ public class VideoMediaProvider extends MediaProvider {
 
     override public function init(itm:PlaylistItem):void {
         setupVideo(itm);
-        // start loading the content only if preload is 'auto'
-        if (itm.preload === "auto") {
```

Comment:
We do not need to check if 'auto' or 'metadata' anymore, because we want to treat them as the same. We do not need to check for 'none', because init is only called if preload !== none from flash.js